src/components/DataDog/DataDog.tsx

```diff
@@ -11,11 +11,11 @@ const DataDog: React.FC = () => {
     ? Array.isArray(query.accountListId)
       ? query.accountListId[0]
       : query.accountListId
-    : '';
+    : null;
 
   const user = session?.user;
   useEffect(() => {
-    if (user) {
+    if (user && accountListId) {
```

Comment:
What if the user is on the accountListId page when they have mulitple accounts. Currently DataDog won't be saving their session

Reply:
You're right, I need to remove that `&& accountListId` condition. Thanks for catching that.